horizons/main.py

```diff
@@ -557,13 +557,10 @@ def edit_game_map(saved_game_name):
 	@return: bool, whether loading succeeded
 	"""
 	saved_games = SavegameManager.get_saves()
-	saved_game_path = _find_map(saved_game_name, saved_games)
-	if not saved_game_path:
+	map_name = _find_map(saved_game_name, saved_games)
+	if not map_name:
 		return False
 
-	accessor = SavegameAccessor(saved_game_path, False)
```

Comment:
The problem with the `accessor` is that if you try to load a savegame map that is not an actual savegame, the `accessor` will complain that the revision is 0 and raise `SavegameTooOld`. Even if this is circumvented/handled, the savegame metadata will still be missing for such a map and loading it will fail. 

On top of this, when I used an actual savegame file (the previous test_savegame.sqlite), it would be parsed by the `accessor` and saved in the database under a different name (either "development" or something else), possibly because as it is now, loading a map into the editor just loads the map's template.

I agree however, that it might be better to `todo_skip` this test or let it fail with the temporary file generated from `NamedTemporaryFile`, rather than modify map loading functions.

I ran GUI tests and the pytest tests after the modification and nothing was explicitly broken.